README.md

```diff
@@ -179,8 +179,12 @@ The userBroker flag setting will allow ADAL to try to call out to the broker.
 Update the AppDelegate.cs file to  include the override method below. This method is invoked everytime the application is launched and is used as an opportunity to process response from the Broker and complete the authentication process. 
 ```C#
 public override bool OpenUrl(UIApplication app, NSUrl url, NSDictionary options)
-{
-    AuthenticationContinuationHelper.SetBrokerContinuationEventArgs(url);
+{            
+	if (AuthenticationContinuationHelper.IsBrokerResponse(sourceApplication))
```

Comment:
What is **sourceApplication**. Is it a member variable ? or did you mean **app** ?